### DISCUSSION TITLE
mspider/user-guide/service-mesh/external-mesh

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2918] [Announcements]
# mspider/user-guide/service-mesh/external-mesh

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-mesh/external-mesh.html

**Comment by nicole-lihui** [comment]
外接网格的截图不对，只需要选择对应命名空间，无需选择版本
